app/indexer/settling/TravelIndexer.ts

```diff
@@ -128,6 +113,32 @@ export default class TravelIndexer extends BaseContractIndexer {
         });
       } catch (e) {}
     }
+    await Promise.allSettled([
+      this.context.prisma.travel.upsert({
+        where: { eventId },
+        create: { ...updates, eventId },
+        update: { ...updates },
+      }),
+      this.saveRealmHistory({
+        realmId: tokenId,
+        bastionId: bastionId,
```

Comment:
how would you distinguish if this is traveling to or from a bastion?

Reply:
this will be shown in the data field: originRealmId or destinationRealmId,  maybe i should have changed this to originId or destinationId but i didnt want to introduce changes that would break the existing client

Reply:
👍